### PR TITLE
&& On exec() and modify push

### PR DESCRIPTION
**Pull request by abkrim**
Hi.

I correct today some lines with command for use properly &&

Also `git push` with `-u` option, is most apropiete that without 

I tested on my owned Gitlab and Github

### CODE REVIEW

File: src/Commands/PublishPackage.php [hunk 1]
```diff
@@ -61,18 +61,18 @@ public function handle()
         $this->conveyor->vendor($this->argument('vendor'));
         $this->conveyor->package($this->argument('name'));
 
-        $this->info('Initialising Git if not already done so...');
```
Comment (Jeroen-G) [inline src/Commands/PublishPackage.php]: Please keep this here.

**Comment by Jeroen-G** [comment]
I don't think you committed anything regarding GitLab?

File: src/Commands/PublishPackage.php (hunk 1 above)
Comment (Jeroen-G) [reply src/Commands/PublishPackage.php]: Github updates the pull request automatically 😄

File: src/Commands/PublishPackage.php [hunk 2]
```diff
@@ -61,18 +61,18 @@ public function handle()
...
+	$this->info('Initialising Git if not already done so...');
```
Comment (Jeroen-G) [inline src/Commands/PublishPackage.php]: Almost there!
Look at the spacing 😉

**Comment by timbroder** [comment]
@Jeroen-G could you cut a release for this, please? Thanks!

**Comment by Jeroen-G** [comment]
Will do!

**Pull request merged**